### ISSUE TITLE
Add support for OS Map API

### ISSUE THREAD

**Issue by JimBacon** [rob-murray/os-leaflet#26]

**Comment by rob-murray** [comment]
Thanks, I had not seen this - hopefully OS Map API is a bit better designed and won't need so many hacks to work with other JS mapping libraries.

**Comment by john-davies** [comment]
Apologies if I'm being thick about this but I find the OS's online offerings extremely confusing! I'm trying to set up a simple example, using http://rob-murray.github.io/os-leaflet/ as a guide, but I'm finding it impossible to get an API key. This is what I've found so far:

1. The page at https://www.ordnancesurvey.co.uk/business-government/tools-support/openspace-support states:

> Existing customers can continue to use OS OpenSpace Free but the product is now closed to new users. If you're looking for a web map, please take a look at our new Zoomstack product.

2. The Zoomstack product doesn't appear to have an API and you have to download the tiles and manage them yourself.

3. The page https://developer.ordnancesurvey.co.uk/os-maps-api-enterprise referenced by @JimBacon above doesn't have a free plan. From a quick check you can't even get a trial key if you're not a company.

From these I'm concluding that it's not possible to set up a new, free API key to use the os-leaflet library. Am I missing something here?

**Comment by rob-murray** [comment]
Yes, looks like it - it doesn't look like this product is still supported. 

I don't actually use this library anymore and I wouldn't recommend any OS products for most web mapping applications.

If for some reason you do need OS mapping in your application, then it's probably best you speak to them and they may be able to suggest what to do.

**Comment by john-davies** [comment]
OK thanks. I was hoping to use the Landranger zoom level but as it's only a hobby project then I'll look at something else.
